functions/Install-DbaMaintenanceSolution.ps1

```diff
@@ -352,7 +376,7 @@ function Install-DbaMaintenanceSolution {
                             IF OBJECT_ID('[dbo].[IndexOptimize]', 'P') IS NOT NULL
                                 DROP PROCEDURE [dbo].[IndexOptimize];
                             ")
-
+                write-host -message $CleanupQuery;
```

Comment:
Do we use ``` Write-Host ``` ???

Reply:
Nope. This line can be removed.

Reply:
or a debug output can be done using `Write-Message`